rnd/autogpt_server/autogpt_server/blocks/llm.py

```diff
@@ -226,7 +246,16 @@ def parse_response(resp: str) -> tuple[dict[str, str], str | None]:
                 if input_data.expected_format:
                     parsed_dict, parsed_error = parse_response(response_text)
                     if not parsed_error:
-                        yield "response", {k: str(v) for k, v in parsed_dict.items()}
+                        yield "response", {
```

Comment:
I think we can utilize type.py for more robust single place conversion method.

Reply:
how would that work here?

Reply:
Are you trying to convert str -> dict[str, list] ?

convert(k, list)

Reply:
i'm tryign to convert json list to list dict

Reply:
pretty sure